### ISSUE TITLE
Replace or improve tmux

### ISSUE THREAD

**Issue by lasse16** [lasse16/dotfiles#8]
I am in dire need of a terminal multiplexer.

As I am using mostly WSL, common tiling window managers do not work. They are superseded by the windows window manager.

Tmux has some severe line wrapping issues, see [this](https://github.com/microsoft/terminal/issues/6987), when using my terminal emulator, WIndows Terminal Preview.

Windows Terminal however encompasses some multiplexer functionality, like pane-splitting for example. It works better than Tmux but does not allow for sessions, including attaching and detaching.

I've been looking at [zellij](https://github.com/zellij-org/zellij).
Apart from the layout-system, their feature set is contained inside the my terminal emulator. At this moment Zellij does not support sessions, hence there is no difference apart from the layout-system between what Windows Terminal Preview and Zellij provide.
Differences between Tmux and Zellij are provided [here](https://github.com/zellij-org/zellij/issues/376).

Three things are are important to me, when looking at terminal multiplexers, apart from simple splitting the terminal:

- Session management (Attaching and detaching for longer running commands)
- Vim integration ( Using the same shortcuts for vim pane splitting and general pane splitting)
- Supporting something similar to [tmux scrollmode](https://www.rockyourcode.com/copy-and-paste-in-tmux/) ( Keyboard only scrolling and selection of terminal output)

A functional layout system is optional as Vim provides something similar already.

**Comment by lasse16** [comment]
Upgraded to tmux-3.2.1

**Comment by lasse16** [comment]
Update:
I switched to alacritty as it is a cross-platform terminal emulator.
Alacritty offers a scrollback too. So, there is still an overlap. By now, I started thinking that there might be no way around it, as almost every terminal emulator implements scrollback. Even alacritty despite wanting to leave all extra features to better suited tools.

Further, I started using tmuxp. A python project which always for storing a tmux window configuration. This is essentially zellij's layout feature.

So, there is no need for me to change anything in this regard anymore.
